rpcs3/headless_application.cpp

```diff
@@ -60,8 +60,7 @@ void headless_application::InitializeCallbacks()
 
 		return false;
 	};
-	callbacks.call_from_main_thread = [this](std::function<void()> func, atomic_t<bool>* wake_up)
-	{
+	callbacks.call_from_main_thread = [this](std::function<void()> func, atomic_t<u32>* wake_up) {
```

Comment:
this is exactly why the clang-format is the way it is right now.

Reply:
I tweaked it for the lesser evil, because otherwise the whole lambda gets shifted right.

Reply:
But that's fine. GalCiv and I weighed the pro's and cons last time we updated the clang-format.
Having the indentation in a lambda or not is both allowed and not nitpicked at the moment.

There are some bugs in clangformat with the inline version.
There are worse format issues that happen in different cases.
I don't remember the details, but I remember that it was easier to use the current settings all things considered.

Reply:
https://github.com/llvm/llvm-project/issues/54827

Reply:
Ok, I'll remove AllowShortLambdasOnASingleLine. The other tweak should be fine though?

Reply:
Idk. I'd have to test with existing code